script/rspec_with_simplecov

```diff
@@ -24,15 +24,19 @@ begin
   # Simplecov emits some ruby warnings when loaded, so silence them.
   old_verbose, $VERBOSE = $VERBOSE, false
 
-  unless ENV['NO_COVERAGE'] || RUBY_VERSION.to_f < 2.1
+  skip_because_no_coverage = ENV['NO_COVERAGE']
+  skip_because_ruby_too_low = RUBY_VERSION.to_f < 2.1
```

Comment:
```suggestion
```
We don’t need this anymore with the ONLY_ON option, do we?

Reply:
We might not! I had been thinking that was a _compatibility_ constraint, but if it was actually just a way to limit when the CI ran simplecov too, then it should probably go.

I actually was a little conflicted here in general, because as it's current written, if someone is _using_ ruby 2.7, the suite would fail locally, since it'll attempt to enforce coverage. The ENV-based approach I used was because I felt like "which ruby runs coverage checks" should be a CI environment feature, rather than something built into the script.. I'd have wanted to just not run coverage by default, and use a `CHECK_COVERAGE` env to turn it _on_, except the script is actually _called_ `rspec_with_simplecov`, so running the script kind of implies that you want to check coverage.

Another option would be to remove the `skip_because_ruby_too_low` check, but add _another_ check that only turns on the `minimum_coverage` if the ruby version is at least 3.1 (which I think is the lowest version we have 100% coverage on).

So I erred on the side of changing _less_, since I wasn't sure which changes to make. Any of those options appeal to you?

Reply:
> ruby version is at least 3.1

This sounds reasonable and simplifies things, also making them similar locally and on CI.